pandas/core/generic.py

```diff
@@ -6835,6 +6840,78 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         Returns
         -------
         shifted : %(klass)s
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'Col1': [10, 20, 30, 20, 15, 30, 45],
+        ...                    'Col2': [13, 23, 33, 23, 18, 33, 48],
+        ...                    'Col3': [17, 27, 37, 27, 22, 37, 52]})
+        >>> print(df)
```

Comment:
Hmm well your DataFrame may be simple enough that you don't need to explicitly print it. That said, if you do need to print you can just say `df` instead of `print(df)`.

I'll leave it as a judgment call on your end if you want this, but if so definitely get rid of the surrounding `print` statement